### ISSUE TITLE
Crashes at openning tab in alesso heist 

### ISSUE THREAD

**Issue by easymoney322** [Kamikaze94/WolfHUD#818]
# Expected Behavior

Tab info popout

# Current Behavior

Crashes 100% time on alesso heist, WHILE HOSTING FROM OFFLINE TROUGH STORY LINE ( no dlc)

## Steps to Reproduce

1. Host Alleso heist trough offline crime.net, using story line option
1.2 ?But all assests trough wolfhud function?
2. Open tab any moment during heist
3. Get crashed

Please notice, that this occur **only during allesso heist**

## Context

Getting crashed with 100% rate. 
First two crashed I've got, I had tabstats disabled in wolfhud settings
The third crash I've got, I had  tabstats on.

* Operating System: Win 10 enterprise 1803
* BLT version:  1.1.6.0  superblt
* Other mods installed: none

## BLT log

<details> 
  <summary>BLT Log</summary>

 
[2020_03_13_log.txt](https://github.com/Kamikaze94/WolfHUD/files/4329095/2020_03_13_log.txt)

</details>

## Crash log(s)

3/3 crashes happened @ 13/03/2020

<details> 
  <summary>Crash.txt</summary>

   
[crashlog.txt](https://github.com/Kamikaze94/WolfHUD/files/4329100/crashlog.txt)

</details>

**Comment by easymoney322** [comment]
UPD:
Assests dont affect crash.
I've had 24h time type in TABSTATS turned on. Disabled it and nothing changed. Still crashes.

**Comment by MEXAHOTABOP** [comment]
can be wrong but looks like bug in payday scripts 

```
string "lib/managers/musicmanager.lua":365: attempt to concatenate field 'current_track' (a nil value)
SCRIPT STACK
recreate_left() @mods/WolfHUD-master/lua/TabStats.lua:105
init() lib/managers/hud/newhudstatsscreen.lua:84
new() core/lib/utils/coreclass.lua:35```

**Comment by ironmandacreep** [comment]
Yeah, definitely seems like it's because you can't select a song during the Alesso heist. That tab is gone from the ready-up screen. I'm having the same exact issue.
